translations/japanese/core-commands.ts

```diff
@@ -0,0 +1,132 @@
+		"That Pokemon is not in your team.": "そのポケモンはあなたのチームの中にはいません。",
+		"View team": "チームを見る",
+		"Must be in a battle room.": "バトルの中でなければいけません。",
+		"This server does not allow offering ties.": "このサーバーでは引き分けの提案が許可されていません。",
+		"You can't offer ties in tournaments.": "トーナメントでは引き分けを提案できません。",
+		"It's too early to tie, please play until turn 100.": "引き分けの提案が早すぎます。100ターンまで待ってください。",
+		"No other player is requesting a tie right now. It was probably canceled.": "他のプレイヤーは引き分けを提案していません。おそらくキャンセルされました。",
+		"${user.name} is offering a tie.": "${user.name}が引き分けを提案しています。",
+		"Accept tie": "引き分けの提案を受け入れる",
+		"Reject": "引き分けの提案を拒否する",
+		"Must be a player to accept ties.": "引き分けを受け入れるにはプレイヤーである必要があります。",
+		"You have already agreed to a tie.": "すでに引き分け同意しています。",
+		"${user.name} accepted the tie.": "${user.name}が引き分けに同意しました。",
+		"All players have accepted the tie.": "全てのプレイヤーが引き分けに同意しました。",
+		"Must be a player to reject ties.": "引き分けを拒否するにはプレイヤーである必要があります。",
+		"${user.name} rejected the tie.": "${user.name}が引き分けを拒否しました。",
+		"This room doesn't have an active game.": "この部屋には、進行中のゲームがありません。",
+		"This kind of game can't be forfeited.": "この種類のゲームは降参できません。",
+		"This game doesn't support /choose": "このゲームでは\"/choose\"が使えません。",
+		"This game doesn't support /undo": "このゲームでは\"/undo\"が使えません。",
+		"You can only save replays for battles.": "リプレイが保存できるのはバトルだけです。",
+		"This battle can't have hidden replays, because the tournament is set to be forced public.": "トーナメントの設定でこのバトルを隠し部屋にすることが許されていません。",
+		"The replay for this battle is already set to hidden.": "このバトルのリプレイはすでに隠されています。",
+		"${user.name} hid the replay of this battle.": "${user.name}がこのバトルのリプレイを非表示にしました。",
+		"You can only do this in battle rooms.": "これはバトル部屋のみで使えます。",
+		"You can only add a Player to unrated battles.": "プレイヤーを追加できるのはレートが有効でないバトルのみです。",
+		"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "プレイヤーは\"p1\"か\"p2\"に設定しなければなりません。\"${target}\"ではありません。",
+		"User ${name} not found.": "ユーザー ${name} は見つかりませんでした。",
+		"User ${name} must be in the battle room already.": "ユーザー\"${name}\"はすでにバトル部屋に入室しています。",
+		"This room already has a player in slot ${target}.": "スロット${name}にはすでにユーザーがいます。",
+		"${targetUser.name} is already a player in this battle.": "\"${targetUser.name}\"はすでにプレイヤーになっています。",
+		"${name} was added to the battle as Player ${playerNum} by ${user.name}.": "",
+		"Player 2": "プレイヤー2",
+		"Players could not be restored (maybe this battle already has two players?).": "プレイヤーの復元ができませんでした。(このバトルにすでに2人プレイヤーがからかもしれません。)",
+		"This game doesn't support /joingame": "このゲームは /joingame が使えません。",
+		"This game doesn't support /leavegame": "このゲームは /leavegame が使えません。",
+		"You can only do this in unrated non-tour battles.": "これはトーナメントでなくかつレートが有効ではないバトルでのみ有効です。",
+		"User ${targetUsername} not found.": "ユーザー ${targetuser.name} は見つかりませんでした。",
+		"${targetUser.name} was kicked from a battle by ${user.name} ${displayTarget}": "",
```

Comment:
```suggestion
		"${targetUser.name} was kicked from a battle by ${user.name} ${displayTarget}": "",
		"${targetUser.name} was kicked from a battle by ${user.name} ${displayReason}": "",
```
also I'll edit this if this was true:
<img src="https://user-images.githubusercontent.com/86363603/148244251-bc5bf105-5519-4bb5-b6ba-8da7f2844261.jpeg" width="300px">

Reply:
(why added 2 lines…)